packages/trader/src/Modules/Account/Sections/Security/ConnectedApps/connected-apps.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+
+class ConnectedApps extends React.Component {
+    state = {
+        is_loading: true,
+    };
+    componentDidMount() {
+        let apps_list = this.props.fetchConnectedApps();
+        apps_list.then(
+            result => {
+                this.props.setConnectedApps(result);
```

Comment:
This can be removed.

Reply:
this one can remove of we do the setConnectedApps inside the fetching function.